test/e2e/app-dir/rsc-basic/rsc-basic.test.ts

```diff
@@ -35,6 +35,7 @@ describe('app dir - rsc basics', () => {
         'styled-components': '6.0.0-beta.5',
         react: 'latest',
         'react-dom': 'latest',
+        'server-only': 'latest',
```

Comment:
server-only is built in and aliased, should be fine if you dont add it

Reply:
We need to test the user installed `node_modules` case. `next/dist` is specially handled in external resolution and I'm trying to simulate the other case here.